index.test-d.ts

```diff
@@ -3,5 +3,13 @@ import exitHook from './index.js';
 
 const unsubscribe = exitHook(() => {}); // eslint-disable-line @typescript-eslint/no-empty-function
 
+const asyncUnsubscribe = exitHook.async({
+	async onExit() {}, // eslint-disable-line @typescript-eslint/no-empty-function
+	minWait: 100,
+});
```

Comment:
I think the API should be:

```js
asyncExitHook({minimumWait: 100}, () => {
	// onExit
});
```

Reply:
`minWait` => `minimumWait`

Reply:
Is there a specific reason we want to lead with the options object? If we do reorder the params, should we also remove the sensible default from `minimumWait` to make it clear you need to decide how long you're willing to wait for shutdown?

Reply:
It's a common convention to have the callback last.

Reply:
> If we do reorder the params, should we also remove the sensible default from minimumWait to make it clear you need to decide how long you're willing to wait for shutdown?

Yeah, I think so. There's really no sensible default. It really depends on what you're doing at shutdown.